### PR TITLE
Update generic_actions.md to seperate Logic Actions from Generic actions

### PR DESCRIPTION
**Pull request by sarjyusuf**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/service_management/workflows/actions_catalog/generic_actions.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ further_reading:
   text: "Learn about integrations"
 ---
 
-Generic actions are workflow actions which are not associated with a Datadog integration or resource. These actions allow you to implement custom logic within your workflow such as branching the workflow based on a condition, making a custom HTTP request, or transforming data and objects with code. As with all workflow actions, you can use the [context variables][1] tab to access any values available in the workflow context.
+Generic actions are workflow actions which are not associated with a tool or integration. This allows you to overcome any action that may not be currently present in the Actions catalog.
```
Comment (rtrieu) [inline content/en/service_management/workflows/actions_catalog/generic_actions.md:15]:
```suggestion
Generic actions are workflow actions that are not associated with a tool or integration. They allow you to overcome any action that may not be currently present in the Actions catalog. 
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Minor feedback, but otherwise looks good!

File: content/en/service_management/workflows/actions_catalog/generic_actions.md [hunk 2]
```diff
@@ -12,7 +12,9 @@ further_reading:
   text: "Learn about integrations"
 ---
 
-Generic actions are workflow actions which are not associated with a Datadog integration or resource. These actions allow you to implement custom logic within your workflow such as branching the workflow based on a condition, making a custom HTTP request, or transforming data and objects with code. As with all workflow actions, you can use the [context variables][1] tab to access any values available in the workflow context.
+Generic actions are workflow actions that are not associated with a tool or integration. They allow you to overcome any action that may not be currently present in the Actions catalog. 
+As with all workflow actions, you can use the [context variables][1] tab to access any values available in the workflow context.
+You can also [request for a new action or entire integration][5].
```
Comment (hestonhoffman) [inline content/en/service_management/workflows/actions_catalog/generic_actions.md:17]:
```suggestion
Generic actions are workflow actions that are not associated with a tool or integration. They allow you to overcome any action that may not be currently present in the Actions catalog.

As with all workflow actions, you can use the [context variables][1] tab to access any values available in the workflow context.

You can also [request a new action or entire integration][5].
```

**Review by rtrieu — APPROVED** [review]
Looks good!

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**